app/main.py

```diff
@@ -16,4 +16,6 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [my_favourite_films, marks, collection_of_coins],
+    "immutable": [lucky_number, pi, one_is_a_prime_number, name, profile_info]}
```

Comment:
The `collection_of_coins` set is categorized under 'mutable'. While sets are mutable in terms of their contents, they are immutable in terms of their structure (i.e., you cannot change the set to a different type). Consider whether this categorization aligns with your intended logic.